index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Como te llamas")
+alert("Hola " + nombreUsuario)
```

Comment:
En este caso me pareceria mejor no usar interpolacion de strings
```suggestion
alert("Hola " + nombreUsuario)
```